tests/openvino/test_modeling.py

```diff
@@ -125,7 +128,10 @@ def test_load_from_hub_and_save_model(self):
         loaded_model = OVModelForSequenceClassification.from_pretrained(self.OV_MODEL_ID, ov_config=ov_config)
         self.assertTrue(manual_openvino_cache_dir.is_dir())
         self.assertGreaterEqual(len(list(manual_openvino_cache_dir.glob("*.blob"))), 1)
-        self.assertEqual(loaded_model.request.get_property("PERFORMANCE_HINT").name, "THROUGHPUT")
+        if get_version() < "2023.3":
```

Comment:
minor comment : it could be replaced with [`is_openvino_version`](https://github.com/huggingface/optimum-intel/blob/c64025dae99596c82fd79cba42b674a1576455f4/optimum/intel/utils/import_utils.py#L192) for clarity
```suggestion
        if is_openvino_version("<", "2023.3"):
```